js/src/forum/components/SignUpModal.js

```diff
@@ -104,6 +104,7 @@ export default class SignUpModal extends Modal {
             className="FormControl"
             name="password"
             type="password"
+            autocomplete="current-password"
```

Comment:
We should use `new-password` here instead.

Reply:
👍🏼